remind.py

```diff
@@ -411,6 +411,10 @@ def to_remind(self, vevent, label=None, priority=None, tags=None, tail=None,
                 remind.append(vevent.dtstart.value.astimezone(self._localtz).strftime('AT %H:%M').replace(' 0', ' '))
             else:
                 remind.append(vevent.dtstart.value.strftime('AT %H:%M').replace(' 0', ' '))
+
+            if posttime:
```

Comment:
I think it would be enough to apply it as an override, regardless of whata is set before.